examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/DataProtectors.java

```diff
@@ -113,11 +106,18 @@
     public PCollectionTuple expand(PCollection<KV<Integer, Row>> inputRows) {
       FailsafeElementCoder<Row, Row> coder =
           FailsafeElementCoder.of(RowCoder.of(schema()), RowCoder.of(schema()));
+
+      Duration maxBuffering = Duration.millis(MAX_BUFFERING);
       PCollectionTuple pCollectionTuple =
-          inputRows.apply(
-              "Tokenize",
-              ParDo.of(new TokenizationFn(schema(), batchSize(), rpcURI(), failureTag()))
-                  .withOutputTags(successTag(), TupleTagList.of(failureTag())));
+          inputRows
```

Comment:
lets name it

Reply:
i mean name transform apply like next one "tokenization"

Reply:
Done, thank you.